pkg/tlsf/contrib/Makefile.include

```diff
@@ -0,0 +1 @@
+UNDEF += $(BINDIR)/tlsf-malloc/tlsf-malloc.o
```

Comment:
I think the `UNDEF` is not necessary for this step. I do not see any `weak` or putting in specific sections.

Reply:
D'oh, that should have gone in a later commit.